### DISCUSSION TITLE
LEITOR DE CODIGO DE BARRAS SELECIONAR PRODUTO

### DISCUSSION THREAD

**Discussion by junimkodi** [RamonSilva20/mapos#1895] [Ideas]
BOA NOITE ESTOU COM O SEGUINTE PROBLEMA.
EU QUERIA SABER SE TEM COMO LER O CODIGO DE BARRAS DO PRODUTO NA HORA DA VENDA E ELE SELECIONAR AUTOMATICAMENTE NO SISTEMA E PREENCHER COM O PRODUTO REFERENTE AO CODIGO DE BARRAS QUE FOI LIDO.
SE PUDER ME AUXILIAR A CONSEGUIR ISSO FICAREI GRATO. OBG

**Comment by mikxingu** [comment]
Bom dia!! Tudo bem? Qual a versão que está usando?

Achei esse comentário interessante, e resolvi fazer um teste.

Cadastrei um produto de teste, com código de barras "123456789".
Ao ler esse código de barras, ele não seleciona automaticamente, mas ele aparece pra selecionar o produto, veja:

### Produto teste
![image](https://user-images.githubusercontent.com/30123586/185415492-cf85c986-927f-455e-b4e0-2c51291ca4cd.png)

Na hora da venda, se eu começar a digitar o código de barras, ele aparece pra mim. Veja:
![image](https://user-images.githubusercontent.com/30123586/185415706-013ed138-71a2-4906-acb5-8ced5efbad9a.png)

Fiz um outro produto só pra ter certeza, e realmente dá certo.
![image](https://user-images.githubusercontent.com/30123586/185416122-b7bc1d17-1740-4a4b-b3a1-de0d1d9fda79.png)

Unica coisa que reparei, e que provavelmente pode acontecer, é que os leitores de código de barras dão um "enter" automaticamente após a leitura. Mas a maioria deles permite que vc configure e tire esse "enter" automático.

Verifique se essa solução te ajuda 👍🏻

**Comment by evertonterra** [comment]
Alguém conseguiu resolver , via código manual encontra , mais ao usar o leitor de código de barras não funciona

**Comment by andremp15** [comment]
Outra coisa interessante após efetuar a venda aparece o número da venda data da venda cliente faturado e ações porém não aparece o nome do produto vendido tem que abrir a venda para poder ver o que vendeu

**Comment by DataPowerTI** [comment]
Estou com esse mesmo problema.
Alguém conseguiu uma solução?
